server.py

```diff
@@ -332,6 +332,7 @@ def chat(chat_request: ChatRequest):
             ai=ai,
             config=config,
             global_instructions=global_instructions,
+            custom_system_prompt=chat_request.custom_system_prompt
```

Comment:
Consider also adding to additional functions:
* `investigate_issues`
* `stream_investigate_issues`
* `issue_conversation`

That way it will be available for RCA as well + follow-up questions on RCA